### PR TITLE
Fix invoke script using dev DB in non-dev

### PR DESCRIPTION
**Pull request by rmarow**

### CODE REVIEW

**Review by MattF-NSIDC — APPROVED** [review]

**Comment by rmarow** [comment]
TODO: Figure out how to resolve the circular import error

**Comment by rmarow** [comment]
Issue: Login manager requires app as an argument.

**Comment by rmarow** [comment]
nearly there -- currently facing google access error.
Screen Shot 2023-09-19 at 5 57 21 PM
```
>>> app.url_map
Map([<Rule '/static/<filename>' (GET, HEAD, OPTIONS) -> static>,
 <Rule '/bootstrap/static/<filename>' (GET, HEAD, OPTIONS) -> bootstrap.static>,
 <Rule '/' (GET, HEAD, OPTIONS) -> root.root>,
 <Rule '/surveys' (GET, HEAD, OPTIONS) -> surveys.view_surveys>,
 <Rule '/new' (GET, POST, HEAD, OPTIONS) -> survey.new_survey>,
 <Rule '/<survey_id>' (GET, HEAD, OPTIONS) -> survey.view_survey>,
 <Rule '/users' (GET, HEAD, OPTIONS) -> users.view_users>,
 <Rule '/login' (GET, HEAD, OPTIONS) -> login.login>,
 <Rule '/google_oauth/google' (GET, HEAD, OPTIONS) -> google.login>,
 <Rule '/google_oauth/google/authorized' (GET, HEAD, OPTIONS) -> google.authorized>,
 <Rule '/response/<survey_id>' (GET, HEAD, OPTIONS) -> response.view_response>])
```

File: docker-compose.dev.yml [hunk 1]
```diff
@@ -23,6 +23,9 @@ services:
       - "USAON_VTA_DB_SQLITE=true"
         # Enable the in-browser debugger: 
       - "FLASK_DEBUG=True"
+      # Creds for talking to Google:
```
Comment (MattF-NSIDC) [inline docker-compose.dev.yml:26]: Let's leave a comment saying that these envvars are only temporary until #144 is resolved?

File: usaon_vta_survey/__init__.py [hunk 2]
```diff
@@ -71,9 +71,11 @@ def before_request():
     from usaon_vta_survey.routes.users import users_bp
 
     app.register_blueprint(users_bp)
-    from usaon_vta_survey.routes.login import blueprint, login_bp
+    from usaon_vta_survey.routes.login import login_bp
 
     app.register_blueprint(login_bp)
+    from usaon_vta_survey.routes.google import blueprint
+
```
Comment (MattF-NSIDC) [inline usaon_vta_survey/__init__.py:88]: I'm guessing the formatter was forcing these blank lines? What about grouping all the imports together, then all the registrations?  Or maybe we could just disable the formatting for these lines. No strong opinion or request to change anything here, just thinking out loud.

File: usaon_vta_survey/routes/login.py [hunk 3]
```diff
@@ -1,23 +1,16 @@
...
-    scope=["profile", "email"],
-)
-
 login_bp = Blueprint('login', __name__, url_prefix='/login')
 
 
 @login_bp.route("")
 def login():
     if not google.authorized:
+        # breakpoint()
```
Comment (MattF-NSIDC) [inline usaon_vta_survey/routes/login.py:13]: Let's get rid of this comment :) 
```suggestion
```

File: usaon_vta_survey/routes/user.py [hunk 4]
```diff
@@ -4,21 +4,19 @@
...
 
-# TODO: figure out how to handle this with app factory
-if envvar_is_true("USAON_VTA_LOGIN_DISABLED"):
-    # HACK: Always logged in as dev user when login is disabled
-    import flask_login.utils as flask_login_utils
-
-    from usaon_vta_survey.util.dev import DEV_USER
-
-    flask_login_utils._get_user = lambda: DEV_USER
+# if envvar_is_true("USAON_VTA_LOGIN_DISABLED"):
```
Comment (MattF-NSIDC) [inline usaon_vta_survey/routes/user.py:8]: Let's leave a comment linking to #144 here as well.

**Review by MattF-NSIDC — APPROVED** [review]
Nice! Just needs a couple comment changes :)

File: usaon_vta_survey/__init__.py (hunk 2 above)
Comment (rmarow) [reply usaon_vta_survey/__init__.py]: went with grouping all the imports.

File: usaon_vta_survey/__init__.py [hunk 5]
```diff
@@ -81,7 +85,11 @@ def before_request():
     app.register_blueprint(login_bp)
     app.register_blueprint(logout_bp)
     app.register_blueprint(blueprint, url_prefix="/google_oauth")
-    app.register_blueprint(response.bp)
+    app.register_blueprint(bp)
```
Comment (MattF-NSIDC) [inline usaon_vta_survey/__init__.py:88]: @rmarow `bp` here is hard to read. What about `response_bp`?

I feel `obs_bp` and `dp_bp` would also be more readable spelling out the words.

**Pull request merged**